targets/ipq40xx-generic

```diff
@@ -96,6 +105,21 @@ device('gl.inet-gl-b1300', 'glinet_gl-b1300', {
 device('linksys-ea6350v3', 'linksys_ea6350v3')
 
 
+-- Meraki
+
+device('meraki-mr33', 'meraki_mr33', {
+	packages = { 
+		-- radio0 is monitoring radio - removed for now
+	    -- the -ct firmware does not have working mesh
+		'-ath10k-firmware-qca9887-ct',
+		'-ath10k-board-qca9887',
+	},
+	factory = false,
+	manifest_aliases = {'meraki-mr33-access-point'},
+	broken = true,  -- open case to install
```

Comment:
Please add _all_ reasons you know of why this device is broken; so that once one is eventually resolved we do not accidentally mark this as supported in the future.

> Please add this device as broken, given the board also bricks the SoC on newer bootloader-versions which is irreversible.

_Originally posted by @blocktrron in https://github.com/freifunk-gluon/gluon/pull/2857#pullrequestreview-1386917707_